core/src/reporter/api.ts

```diff
@@ -0,0 +1,41 @@
+import axios from 'axios'
+import { Logger } from 'pino'
+import { ORAKL_NETWORK_API_URL } from '../settings'
+import { buildUrl } from '../utils'
+import { OraklError, OraklErrorCode } from '../errors'
+import { ISubmissionData } from './types'
+
+export async function storeSubmission({
+  submissionData,
+  logger
+}: {
+  submissionData: ISubmissionData
+  logger?: Logger
+}) {
+  try {
+    const endpoint = buildUrl(ORAKL_NETWORK_API_URL, `last-submission/upsert`)
```

Comment:
```suggestion
    const endpoint = buildUrl(ORAKL_NETWORK_API_URL, `last-submission`)
    const response = await axios.put(endpoint, { ...submissionData })
```

It is better to avoid an explicit definition of what kind of operation happens when calling a certain API endpoint. In this case it would be better to use `PUT` HTTP method.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods

> The PUT method replaces all current representations of the target resource with the request payload.